client/my-sites/stats/grid-layout.scss

```diff
@@ -125,10 +124,6 @@ $grid-vertical-gutters: 32px;
 		7
 	);
 
-	@media (max-width: 660px) {
-		padding: 0 16px;
```

Comment:
While testing the changes, I noticed that this CSS was adding unnecessary padding not present for the same type of grid on the Traffic page.